lab-carolina/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lab-01-modular-patterns",
+  "version": "1.0.0",
+  "description": "![CF](assets/shield-32x32.png) Lab 01: Modular Patterns and Testing ===",
```

Comment:
Might consider changing the description? Be sure to give a thorough review of the `npm init` process as it's moving you through the setup.